include/boost/beast/core/detail/bind_handler.hpp

```diff
@@ -155,11 +155,12 @@ class bind_wrapper
 
     template<class Function>
     friend
-    void asio_handler_invoke(
+    boost::asio::asio_handler_invoke_is_deprecated
+    asio_handler_invoke(
         Function&& f, bind_wrapper* op)
     {
         using net::asio_handler_invoke;
```

Comment:
nit: These should say `boost::asio::asio_handler_invoke` not `net::asio_handler_invoke`, because this hook is not part of the TS and never will be.